eZ/Bundle/EzPublishCoreBundle/Command/DebugConfigResolverCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+class DebugConfigResolverCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}.
+     */
+    public function configure()
+    {
+        $this->setName('ezplatform:debug:config-resolver');
+        $this->setAliases(['ezplatform:debug:config']);
+        $this->setDescription('Debug / Retrive parameter from Config Resolver');
+        $this->addArgument(
+            'parameter',
+            InputArgument::REQUIRED,
+            'The configuration resolver parameter to return, for instance "languages" or "http_cache.purge_servers"'
+        );
+        $this->addOption(
+            'json',
```

Comment:
In general I feel we should have here what `debug:container` command from Symfony offers: 
```
      --format=FORMAT            The output format (txt, xml, json, or md) [default: "txt"]
```

Reply:
It started out as `--oneline`, raw output without newlines in order to use in scripts. @adamwojs asked to complicate it, now you want more :)

Can we just drop the option and you guys can add whatever you'd like later? ;)

Reply:
@alongosz `--oneline` or drop it?

Reply:
> @alongosz `--oneline` or drop it?

After digging a bit... let's keep `json`, it might be actually useful. For what I've asked for Symfony uses `\Symfony\Bundle\FrameworkBundle\Console\Helper\DescriptorHelper` which is marked as `@internal`, so I'm withdrawing my request.